src/views/LoginView.js

```diff
@@ -1,58 +1,48 @@
 'use strict'
 
-import React from 'react'
+import React, { lazy } from 'react'
 import { hot } from 'react-hot-loader'
 import { Redirect } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import { observer } from 'mobx-react'
 import { withTranslation } from 'react-i18next'
 import { CSSTransitionGroup } from 'react-transition-group'
-import LoadAsync from '../components/Loadable'
 
 import { version } from '../../package.json'
 
 import Logger from '../utils/logger'
 
-import RootStoreContext from '../context/RootStoreContext'
+import RootContext from '../context/RootContext'
 
 import '../styles/LoginView.scss'
 
-const BackgroundAnimation = LoadAsync({
-  loader: () =>
-    import(/* webpackChunkName: "BackgroundAnimation" */ '../components/BackgroundAnimation')
-})
-const LoginForm = LoadAsync({
-  loader: () => import(/* webpackChunkName: "LoginForm" */ '../components/LoginForm')
-})
+const BackgroundAnimation = lazy(() =>
+  import(/* webpackChunkName: "BackgroundAnimation" */ '../components/BackgroundAnimation')
+)
+
+const LoginForm = lazy(() => import(/* webpackChunkName: "LoginForm" */ '../components/LoginForm'))
 
 const logger = new Logger()
 
 class LoginView extends React.Component {
```

Comment:
Couldn't this be a functional component as well?

Reply:
I left it because there was no need to touch. When that component requires changes it's better to refactor it then.